api/models/Item.cs

```diff
@@ -44,6 +44,7 @@ public int PackOrderQuantity
     // Foreign Key Relationship
     public Guid? ItemGroupId { get; set; }
     public ItemGroup? ItemGroup { get; set; }
+    public Guid? SupplierId { get; set; }
```

Comment:
Je moet nog dit required maken want een item kan niet bestaan zonder supplier
```suggestion
    [Required]
    public Guid? SupplierId { get; set; }
```